### DISCUSSION TITLE
press/soft/daily/%E4%B8%9C%E6%96%B9%E5%81%A5%E5%BA%B7%E8%86%B3%E9%A3%9F%E6%A8%A1%E5%BC%8F

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#2] [General]
# press/soft/daily/%E4%B8%9C%E6%96%B9%E5%81%A5%E5%BA%B7%E8%86%B3%E9%A3%9F%E6%A8%A1%E5%BC%8F

A hub to archive thinkings

https://plantree.github.io/press/soft/daily/%E4%B8%9C%E6%96%B9%E5%81%A5%E5%BA%B7%E8%86%B3%E9%A3%9F%E6%A8%A1%E5%BC%8F.html

**Comment by kequeen** [comment]
所以你现在已经在尝试了么
